### DISCUSSION TITLE
Batteriestatus in Wiki

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#5] [Q&A] [answered]
Gibt es irgendwo eine Tutorial für den Batterie Status mit Icons und Textfarbe?

Sprich von 
0% - 20% Das Icon und die Textfarbe
21% - .... usw.

**Comment by mcuiobroker** [comment, marked as answer]
Ich habe mal die grundsätzlichen Eingabemöglichkeiten für die Felder hier zusammengefasst:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/geraet/eingabewerte-fuer-die-geraete-dp-state-felder
Ansonsten die Suche nutzen.
Vieles funktionierte ja auch schon in v2 und ist somit dort erklärt.

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/styles/effekte/svg-animation-aus-icons

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/styles/effekte

**Reply by Mephisto090** [reply]
```
{
  "<10%":"flat-color-icons:empty-battery",
  "<50%":"flat-color-icons:low-battery",
  "<80%":"flat-color-icons:middle-battery",
  "<90%":"flat-color-icons:high-battery",
  "default":"flat-color-icons:full-battery"
}
```

Das Beispiel kann gerne aufgenommen werden

**Reply by mcuiobroker** [reply]
Es wird immer dann der Zustand gewählt, wenn der erste richtige Wert gefunden wird von jarvis.
Also bei dir <50% -> 49% ist bei dir LOW Battery?

**Reply by Mephisto090** [reply]
Ja genau so ist das gedacht :)

> Es wird immer dann der Zustand gewählt, wenn der erste richtige Wert gefunden wird von jarvis.

Das sollte auf jedenfall ins Wiki

**Reply by mcuiobroker** [reply]
Steht drin:
![image](https://user-images.githubusercontent.com/84818934/193448399-6c58de27-b37d-42c5-b87e-0b15ae70c8c8.png)
